flakes/k8s-controller-tools/flake.nix

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
We can also contribute upstream like this:
https://github.com/NixOS/nixpkgs/pull/288288

But this already unblocks us regardless.

Reply:
I strongly advise to contribute upstream, yes.

Reply:
I will give that a try next. The change itself is small, but the barriers to first contribution might not be so trivial. Let's see.

Reply:
Upstream contribution in NixOS packages repo:
[PR #338425 kubernetes-controller-tools: 0.14.0 -> 0.16.1](https://github.com/NixOS/nixpkgs/pull/338425)